R/zchunk_L241.trade_input.R

```diff
@@ -0,0 +1,193 @@
+    A_demand_technology_R %>%
+      filter(supplysector == "FoodDemand_Crops") %>%
+      left_join_error_no_match(L241.ag_kcalg_R_C_Yf.mlt, by = c("region" = "region", "technology" = "GCAM_commodity")) %>%
+      mutate(coefficient = LOCAL_FOOD_FRACT / round(value, aglu.DIGITS_CALOUTPUT),
+             minicam.energy.input = "ag_trade") %>%
+      select(-value) %>%
+      repeat_add_columns(tibble(year = aglu_demand_futureyears)) %>%
+      select(one_of(aglu.NAMES_STUBTECHCOEF_NM)) ->
+      L241.StubAgTradeCoeff_food
+
+    # Coefficient for ag_trade of non-food crops (incl secondary products) (71-76)
+    A_demand_technology_R %>%
+      filter(supplysector == "NonFoodDemand_Crops") %>%
+      mutate(coefficient = LOCAL_FOOD_FRACT,
+             minicam.energy.input = "ag_trade") %>%
+      repeat_add_columns(tibble(year = aglu_demand_futureyears)) %>%
+      select(one_of(aglu.NAMES_STUBTECHCOEF_NM)) ->
+      L241.StubAgTradeCoeff_nonfood
+
+    # Coefficient for ag_trade of feed crops (incl secondary products) (78-83)
+    A_an_input_technology_R %>%
+      filter(supplysector == "FeedCrops") %>%
+      mutate(coefficient = LOCAL_FOOD_FRACT,
+             minicam.energy.input = "ag_trade") %>%
+      repeat_add_columns(tibble(year = aglu_demand_futureyears)) %>%
+      select(one_of(aglu.NAMES_STUBTECHCOEF_NM)) ->
+      L241.StubAgTradeCoeff_feed
+
+    # Output coefficient for RES ag_trade of food crops (incl secondary products) (85-90)
+    L2012.AgProduction_ag_irr_mgmt %>%
+      filter(year == max(BASE_YEARS)) %>%
+      select(-year) %>%
+      repeat_add_columns(tibble(year = aglu_demand_futureyears)) %>%
+      mutate(res.secondary.output = "ag_trade",
+             output.ratio = 1) %>%
+      select(one_of(aglu.NAMES_AGRES)) ->
+      L241.AgProdTech_RES_output
+
+    # Market for policy portfolio (92-100)
+    GCAM_region_names %>%
+      mutate(policy.portfolio.standard = "ag_trade",
+             market = "Med_to_High_Income",
+             market = if_else(region %in% L241.low_reg, region, market),
+             policyType = "RES") %>%
+      repeat_add_columns(tibble(year = aglu_demand_futureyears)) %>%
+      mutate(constraint = 1) %>%
+      select(-GCAM_region_ID) ->
+      L241.RES_Market
+
+    # Remove any regions for which agriculture and land use are not modeled
+    L241.StubAgTradeCoeff_food    <- filter(L241.StubAgTradeCoeff_food, !region %in% aglu.NO_AGLU_REGIONS)
+    L241.StubAgTradeCoeff_nonfood <- filter(L241.StubAgTradeCoeff_nonfood, !region %in% aglu.NO_AGLU_REGIONS)
+    L241.StubAgTradeCoeff_feed    <- filter(L241.StubAgTradeCoeff_feed, !region %in% aglu.NO_AGLU_REGIONS)
+    L241.AgProdTech_RES_output    <- filter(L241.AgProdTech_RES_output, !region %in% aglu.NO_AGLU_REGIONS)
+    L241.RES_Market               <- filter(L241.RES_Market, !region %in% aglu.NO_AGLU_REGIONS)
+
+    # Produce outputs
+    L241.StubAgTradeCoeff_food %>%
+      add_title("descriptive title of data") %>%
```

Comment:
@pkyle Here and below could use help on an English description of what these datasets are, etc.